gax/test/test-application/src/index.ts

```diff
@@ -657,7 +660,6 @@ async function testServerStreamingRetrieswithRetryRequestOptions(
   const finalData: string[] = [];
   const retryRequestOptions = {
     objectMode: true,
-    retries: 1,
     maxRetryDelay: 70,
```

Comment:
```suggestion
    retries: 2,
    maxRetryDelay: 70,
```
I am not surprised that this test is broken - reading the comment that says it should retry twice but yet having retries set to 1 is bonkers! Who wrote this?? (me, I did 🤦🏻‍♀️ ) I think instead though, that retries should be set to 2 - that way we are actually counting retries properly, reflecting your code changes, but still preserving what this test does, which is makes sure that retry behavior is the same even if we use RetryRequestOptions (which get converted under the hood) instead of RetryOptions